docs/cumulusci_flow.rst

```diff
@@ -0,0 +1,273 @@
+To alleviate this pain point, CumulusCI can ensure that all release branches propagate commits they receive to other existing release branches that correspond to future releases.
+
+Consider the following branches in a GitHub repository:
+
+   * ``main`` - Source of Truth for Production
+   * ``feature/002`` - The next major production release
+   * ``feature/002__feature1`` - A single feature associated with release ``002``
+   * ``feature/002__large_feature`` - A large feature associated with release ``002``
+   * ``feature/002__large_feature__child1`` - First chunk of work for the large feature
+   * ``feature/002__large_feature__child2`` - Second chunk of work for the large feature
+   * ``feature/003`` - The release that comes after ``002``
+   * ``feature/003__feature1`` - A single feature associated with release ``003``
+
+In this scenario, CumulusCI ensures that when ``feature/002`` receives a commit, that that commit is also merged into ``feature/003``.
+This kicks off tests in our CI system and ensures that functionality going into ``feature/002`` doesn't break work being done for future releases.
+Once those tests pass, the commit on ``feature/003`` is merged to ``feature/003__feature1`` because they adhere to the parent/child naming convention described above.
+Commits **never** propagate in the opposite direction. (A commit to ``feature/002`` would never be merged to ``feature/001`` if it was an existing branch in the GitHub repository).
+
+**Propagating commits to future release branches is turned off by default.** 
+If you would like to enable this feature for your GitHub repository, you can set the ``update_future_releases`` option on the ``github_automerge_feature`` task in your ``cumulusci.yml`` file: 
+
+.. code-block:: yaml 
+
+   tasks:
+      github_automerge_feature:
+      options:
+         update_future_releases: True
+
+
+
+Orphan Branches
+^^^^^^^^^^^^^^^
+If you have both a parent and a child branch, and the parent is deleted, this creates an orphaned branch.
+Orphaned branches do not receive any auto-merges from any branches.
+You can rename an orphaned branch to include the ``feature/`` prefix and contain no double underscores ('__') to begin receiving merges from the main branch again.
+
+If we have a parent and child branch: ``feature/myFeature`` and ``feature/myFeature__child``, and ``feature/myFeature`` (the parent) is deleted, then ``feature/myFeature__child`` would be considered an orphan.
+Renaming ``feature/myFeature__child`` to ``feature/child`` will allow the orphan to begin receiving automerges from the main branch.
+
+
+
+
+CumulusCI Flow vs Github Flow
+-----------------------------
+Since CumulusCI Flow is largely an extension of Github Flow, the differences are mostly additional process in CumulusCI Flow that's not in Github Flow:
```

Comment:
```suggestion
Since CumulusCI Flow is largely an extension of GitHub Flow, the differences are mostly additional processes in CumulusCI Flow that help make it more effective for large-scale Salesforce projects:
```